content/en/docs/tasks/job/pod-failure-policy.md

```diff
@@ -28,42 +28,50 @@ You should already be familiar with the basic use of [Job](/docs/concepts/worklo
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-## Using Pod failure policy to avoid unnecessary Pod retries
+## Usage scenarios
+
+We consider the following usage scenarios for the Job's _Pod failure policy_ feature:
```

Comment:
In docs, we [avoid using "we"](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we)
```suggestion
Consider the following usage scenarios for Jobs that define a Pod failure policy :
```

No need for italics as this is not the first use of the term.